crates/proof-of-sql/src/sql/logical_plans/mod.rs

```diff
@@ -2,6 +2,7 @@
 mod error;
 pub use error::LogicalPlanError;
 mod expr;
-pub use expr::Expr;
+pub use expr::{BinaryOperator, Expr};
 mod plan;
 pub use plan::LogicalPlan;
+mod sqlparser;
```

Comment:
I feel like this can be confusing.
Perhaps `ast_to_plan`, or `sqlparser_conversions`.
I don't have great alternative.

Reply:
I think `ast_to_plan` is good. If this becomes more complex a few months from now we can then split it.